MvvmCross/iOS/iOS/Views/Presenters/MvxIosViewPresenter.cs

```diff
@@ -111,48 +111,43 @@ public virtual void Show(IMvxIosView view, MvxViewModelRequest request)
             MvxRootPresentationAttribute attribute,
             MvxViewModelRequest request)
         {
+            // check if viewController is trying to initialize a navigation stack
+            if (attribute.WrapInNavigationController)
+            {
+                var navigationController = CreateNavigationController(viewController);
+                MasterNavigationController = navigationController as MvxNavigationController;
```

Comment:
@mvanbeusekom  mmm, l am not clear this logic, but if you were right, the `TabBarViewController = tabBarController;` also need call `CloseTabBarViewController();` and  the `SplitViewController ` is same.  @nmilcoff  please give some suggstion.